openedx/features/edly/api/v1/urls.py

```diff
@@ -1,8 +1,14 @@
 from rest_framework import routers
 
+from openedx.features.edly.api.v1.views.course_enrollments import EdlyCourseEnrollmentViewSet
 from openedx.features.edly.api.v1.views.user_sites import UserSitesViewSet
 
 router = routers.SimpleRouter()
 router.register(r'user_sites', UserSitesViewSet, base_name='user_sites')
+router.register(
+    r'course_enrollment',
+    EdlyCourseEnrollmentViewSet,
+    base_name='course_enrollment'
+    )
```

Comment:
```
    base_name='course_enrollment`,
)

Reply:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/86109301/157668600-c2fb3f13-f47f-4d02-be8c-fe0d6633b685.png">

Keeping it consistent with above. Also I don't think it is necessary as it wont reduce any git diff either.